.github/actions/setup-build/action.yml

```diff
@@ -26,13 +32,15 @@ runs:
 
   - name: Install PyTorch nightly depends
     run: |
-      python -m pip install -r pytorch-requirements.txt
+      python -m pip install -r pytorch-${{ inputs.torch-version }}-requirements.txt
       python -m pip install -r build-requirements.txt
     shell: bash
 
   - name: Install prerequisites (Linux)
     if: ${{ runner.os == 'Linux' }}
-    run: sudo apt-get install --yes ccache ninja-build
+    run: |
+      sudo apt-get update
+      sudo apt-get install --yes ccache ninja-build
```

Comment:
these should already be in the builder lets remove it.

Reply:
ccache is still needed, so I will not fully remove it

Reply:
The CI runner already has it.